gpii/node_modules/testing/src/RunTestDefs.js

```diff
@@ -0,0 +1,240 @@
+/*
+
+    A collection of functions to expand "test defs" and run them as tests.
```

Comment:
Add a little more strategy hint explaining the kinds of manipulation these do in this particular context - e.g. that they pick particular mixin grades for the environment and a core sequence grade, and sketch the kinds of variation that they deal with (servers with Couch, etc.)

Reply:
I added a summary with a reference to even more details later in the file.